### DISCUSSION TITLE
Can I share the same node db?

### DISCUSSION THREAD

**Discussion by laozhang713** [Chia-Network/chia-blockchain#10010] [Support]
I need to open multiple chia wallets in the LAN for personal reasons. I would like to ask if there is a way to share the same node db.
The way I use the shared file to modify the config db directory, only one wallet can be opened, the rest cannot be opened.

harvester does not meet my needs.

**Comment by shaneo257** [comment]
have you thought about doing this?
https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines
